docs/installation.rst

```diff
@@ -44,26 +44,29 @@ We also recommend using the subsetting tools in `clisops`_ (see: :doc:`clisops.c
 `xclim` is regularly tested against the main development branches of a handful of key base libraries (xarray, cftime, flox, pint).
 For convenience, these libraries can be installed alongside `xclim` using the following pip-installable recipe:
 
-.. code-block::
+.. code-block:: shell
 
     $ pip install -r requirements_upstream.txt
-    # Or, alternatively:
+
+Or, alternatively:
+
+.. code-block:: shell
+
     $ make upstream
 
 .. _flox: https://github.com/xarray-contrib/flox
 .. _clisops: https://github.com/roocs/clisops
 
 Another optional library is `SBCK`_, which provides experimental adjustment methods to extend :doc:`xclim.sdba <sdba>`.
 It can't be installed directly from pip or conda and has one complex dependency : `Eigen`_.
-Please refer to Eigen's and SBCK's docs for the recommended installation instructions.
-However, Eigen is available on conda, so one can do:
+Please refer to Eigen's and SBCK's docs for the recommended installation instructions. However, Eigen is available on conda, so one can do:
 
-.. code-block::
+.. code-block:: shell
 
    $ conda install -c conda-forge eigen pybind11
    $ pip install "git+https://github.com/Ouranosinc/SBCK.git@easier-install#egg=sbck&subdirectory=python"
```

Comment:
```suggestion
   $ pip install "git+https://github.com/yrobink/SBCK.git@master#egg=sbck&subdirectory=python"
```
The branch has been merged some time ago, we can use this PR to update the doc too!

Reply:
This would require a change to the `opt-slow` workflow too, I couldn't suggest it as it was out of reach.